src/Truncate.js

```diff
@@ -188,7 +191,8 @@ export default class Truncate extends Component {
 
         const lines = [];
         const text = innerText(elements.text);
-        const textLines = text.split('\n').map(line => line.split(' '));
+        const wordSeparator = breakAll ? '' : ' ';
+        const textLines = text.split('\n').map(line => line.split(wordSeparator));
```

Comment:
Instead of having a `breakAll` property, we could introduce a `separator` property, that defaults to `' '` and use it here. What do you think?